cell_tree2d/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record record.txt
```

Comment:
I"ve taken to using pip intsead:

pip install .

I'm pretty sure that pip injects setuptools and those flags in, but also ensures that the all the pip-friendly meta-data will be there, now ans in the future.

I **think** that's the "recommended" way to do it now.

(this will also work with packages that are using raw distutils...)